src/components/seo.jsx

```diff
@@ -76,6 +76,11 @@ export function Seo({
         sizes="180x180"
         href="/apple-touch-icon.png"
       />
+
+      <script
+        src={`${process.env.ONWARD_WIDGET_URL}?v=${Date.now()}`}
```

Comment:
using an env variable lets me change it in dev. The `Date.now()` is for cache busting.